impl/src/main/java/org/apache/myfaces/config/impl/FacesConfigUnmarshallerImpl.java

```diff
@@ -179,18 +180,8 @@ public FacesConfigImpl getFacesConfig(InputStream in, String systemId) throws IO
         {
             throw new FacesException(e);
         }
-        finally
-        {
-            try
-            {
-                in.close();
-            }
-            catch (IOException e)
-            {
-                // ignore silently
-            }
-        }
-          
+        // ignore silently
```

Comment:
I don't know if you need these trailing comments?

Reply:
done